backend/functions/disconnect.ts

```diff
@@ -1,5 +1,30 @@
-import { success } from '../libs/response';
+import { APIGatewayProxyResult } from 'aws-lambda';
+import { dynamoDB } from '../libs/dynamodb-utils';
+import { WatcherDisconnects } from '../../extension/src/communications/from-extension-to-server';
+import { IEvent, success } from '../libs/response';
+import { findAndEnsureRoomJoined, leaveRoom } from '../libs/room-operations';
+import { Room } from '../../extension/src/types';
+
+export const main = async (event: IEvent): Promise<APIGatewayProxyResult> => {
+    const disconnectEvent: WatcherDisconnects = JSON.parse(
+        event.body,
+    ) as WatcherDisconnects;
+    const { roomId } = disconnectEvent;
```

Comment:
How come the room if it's present here? Shouldn't we get at best the connection I'd ?